src/Symfony/Bridge/Twig/Tests/Command/DebugCommandTest.php

```diff
@@ -137,7 +138,7 @@ public function getDebugTemplateNameTestData()
   (None)      templates%e%A
   %A
   @Twig       templates/bundles/TwigBundle%e%A
-              vendors/twig-bundle/Resources/views%e%A 
```

Comment:
All whitespace removal should be reverted

Reply:
Reverted.

Reply:
I can still see many whitespace removal. Maybe your IDE is doing it automatically.